src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -6161,7 +6172,7 @@ namespace InternalHeatGains {
         Real64 OperSchedFrac;                             // Operating schedule fraction
         Real64 CPULoadSchedFrac;                          // CPU loading schedule fraction
         Real64 AirConnection;                             // Air connection type
-        Real64 TSupply;                                   // Supply air temperature [C]
+        Real64 TSupply(0.0);                              // Supply air temperature [C]
```

Comment:
Applying clang format resulted in quite a lot of changes, but this is one of the functional changes.